scout/server/blueprints/cases/controllers.py

```diff
@@ -978,6 +980,87 @@ def vcf2cytosure(store, institute_id, case_name, individual_id):
     return (individual_obj["display_name"], individual_obj["vcf2cytosure"])
 
 
+def phenopacket_hpo(case):
```

Comment:
Perhaps, it will be a very thin extension. But I agree the case is a busy unit; perhaps one could at least divide the "controller" into several categories in separate files. I'll check.

Reply:
Actually, yes, an extension will make a lot of sense in terms of interacting with an external phenotype backend service,  which is the natural next step for this anyway. And why not have it convert files as well. I'll try that and see what fits!